client/my-sites/customer-home/cards/tasks/notice-special-20-coupon/index.js

```diff
@@ -0,0 +1,26 @@
+import announcementImage from 'calypso/assets/images/marketplace/notice-special-20-coupon.svg';
+import { NOTICE_SPECIAL_20_COUPON } from 'calypso/my-sites/customer-home/cards/constants';
+import Task from 'calypso/my-sites/customer-home/cards/tasks/task';
+
+const NoticeSpecial20Coupon = () => {
+	const description = (
+		<>
+			Free domain for one year, plugins & themes, live chat support and more. Use <b>Special20</b>{ ' ' }
+			at checkout. Coupon expires on <b>June 30th.</b>
```

Comment:
I belive this coupon will be running every June 30th.

Reply:
Can we add as a comment the relevant P2 / wpcom code? Or generally some commenting around the expiring? 

For example, adding this code now so that it runs in 28th of June 2024, run for 48 hours, and expire in 30th of June 2024 feels weird.

Reply:
@victorespigares  -- Maybe you can clear this one up? :)

Reply:
Just a heads up that we're discussing how to move this initiative within the ML sales umbrella. More context in pbxNRc-2Tw-p2#comment-4745

Reply:
I've provided D117246#2350888-code in regards to this ML Sales-centric shift indirection, if it doesn't increase the scope or is desired.